common/services/prismic/parsers.js

```diff
@@ -544,6 +547,39 @@ export function parseOnThisPage(fragment: PrismicFragment[]): Link[] {
     });
 }
 
+export function parseMediaObjectLinks(
+  fragment: PrismicFragment[]
+): Array<MediaObjectType> {
+  const cannotParse = 'could_not_parse';
+  // filter media_object
+  const filteredMediaObject = fragment.filter(mediaObject => {
+    if (mediaObject.content?.type === 'media_object') {
```

Comment:
[See here](https://github.com/wellcomecollection/wellcomecollection.org/pull/5650/files#r512092393)

Reply:
after through from regression testing on prismic ... i was able to add a linked document without a media_object. (basically a blank one )